website/print/render/racer.inc

```diff
@@ -97,7 +102,11 @@ if (!is_subclass_of(document_class(), 'PrintableRacerDocument')) {
       if (isset($_GET['where'])) {
         $sql = $sql.' WHERE '.$_GET['where'];
       }
-      $sql = $sql.' ORDER BY lastname, firstname, carnumber';
```

Comment:
This whole 'else' clause is something of an unfinished thought.  When invoked from print.php, $_GET['ids'] is always set, so we never get here anyway.  I'm not super-enthusiastic about building it out unless you have a use case I'm not seeing.

Reply:
Sorry I didn't realize that the ids now always are set -- I agree that this is a better solution so I have reverted these changes.